shell/common/shell.cc

```diff
@@ -816,6 +816,12 @@ void Shell::OnPlatformViewSetViewportMetrics(const ViewportMetrics& metrics) {
   FML_DCHECK(is_setup_);
   FML_DCHECK(task_runners_.GetPlatformTaskRunner()->RunsTasksOnCurrentThread());
 
+  if (metrics.device_pixel_ratio <= 0) {
+    FML_DLOG(ERROR)
```

Comment:
Why DLOG instead of LOG?  Most developers build in --release because that is the default configuration for the build scripts.  Making this a LOG would greatly aid in debugging

Reply:
We try to avoid logs that are not actionable by users. I agree this is useful for debugging, but this message would likely confuse users, and there's no action they can take to fix it.

Reply:
Ahh makes sense, I need to start using --unopt to build with